documentation/modules/metrics/ref_metrics-config-files.adoc

```diff
@@ -5,50 +5,113 @@
-│   ├── strimzi-kafka.json
-│   ├── strimzi-zookeeper.json
+│   ├── strimzi-kafka-exporter.json
 │   ├── strimzi-kafka-mirror-maker-2.json
+│   ├── strimzi-kafka.json
 │   ├── strimzi-operators.json
-│   ├── strimzi-kafka-bridge.json
-│   ├── strimzi-cruise-control.json
-│   └── strimzi-kafka-exporter.json <3>
-├── kafka-connect-metrics.yaml <4>
-├── kafka-metrics.yaml <5>
-├── kafka-mirror-maker-2-metrics.yaml <6>
-├── kafka-bridge-metrics.yaml <7>
-├── kafka-cruise-control-metrics.yaml <8>
+│   └── strimzi-zookeeper.json
+├── grafana-install
+│   └── grafana.yaml <2>
 ├── prometheus-additional-properties
-│   └── prometheus-additional.yaml <9>
+│   └── prometheus-additional.yaml <3>
 ├── prometheus-alertmanager-config
-│   └── alert-manager-config.yaml <10>
-└── prometheus-install
-    ├── alert-manager.yaml <11>
-    ├── prometheus-rules.yaml <12>
-    ├── prometheus.yaml <13>
-    ├── strimzi-pod-monitor.yaml <14>
+│   └── alert-manager-config.yaml <4>
+├── prometheus-install
+│    ├── alert-manager.yaml <5>
+│    ├── prometheus-rules.yaml <6>
+│    ├── prometheus.yaml <7>
+│    ├── strimzi-pod-monitor.yaml <8>
+├── kafka-bridge-metrics.yaml <9>
+├── kafka-connect-metrics.yaml <10>
+├── kafka-cruise-control-metrics.yaml <11>
+├── kafka-metrics.yaml <12>
+└── kafka-mirror-maker-2-metrics.yaml <13>
 --
-<1> Installation file for the Grafana image
-<2> Grafana dashboards
-<3> Grafana dashboard specific to xref:assembly-kafka-exporter-{context}[Kafka Exporter]
-<4> Metrics configuration that defines Prometheus JMX Exporter relabeling rules for Kafka Connect
-<5> Metrics configuration that defines Prometheus JMX Exporter relabeling rules for Kafka and ZooKeeper
-<6> Metrics configuration that defines Prometheus JMX Exporter relabeling rules for Kafka Mirror Maker 2.0
-<7> Kafka Bridge resource with metrics enabled
-<8> Metrics configuration that defines Prometheus JMX Exporter relabeling rules for Cruise Control
-<9> Configuration to add roles for service monitoring
-<10> Hook definitions for sending notifications through Alertmanager
-<11> Resources for deploying and configuring Alertmanager
-<12> Alerting rules examples for use with Prometheus Alertmanager (deployed with Prometheus)
-<13> Installation file for the Prometheus image
-<14> Prometheus job definitions to scrape metrics data from pods
+<1> Grafana dashboards.
+<2> Installation file for the Grafana image.
+<3> Additional configuration to scrape metrics for CPU, memory and disk volume usage, which comes directly from the Kubernetes cAdvisor agent and kubelet on the nodes.
+<4> Hook definitions for sending notifications through Alertmanager.
+<5> Resources for deploying and configuring Alertmanager.
+<6> Alerting rules examples for use with Prometheus Alertmanager (deployed with Prometheus).
+<7> Installation resource file for the Prometheus image.
+<8> PodMonitor definitions that translate to jobs for the Prometheus operator to be able to scrape metrics data directly from pods.
```

Comment:
Maybe I wasn't clear. `PodMonitor` is a Prometheus operator concept. When it's applied, then the Prometheus operator translates it in a "job configuration" that is Prometheus (server) concept.
So maybe something like:

`PodMonitor` definitions translated by the Prometheus operator into jobs for the Prometheus server to be able to scrape metrics data directly from pods.

Reply:
Thanks Paolo. I've updated as you suggest